src/h5web/visualizations/shared/AxisSystemProvider.tsx

```diff
@@ -1,28 +1,24 @@
 import React, { createContext, ReactElement, ReactNode } from 'react';
-import { AxisDomains } from './models';
+import { AxisConfig } from './models';
 
 export interface DisplayAxisProps {
-  axisDomains: AxisDomains;
-  showGrid?: boolean;
-  hasXLogScale?: boolean;
-  hasYLogScale?: boolean;
```

Comment:
The log setting is no longer bound to x and y anymore. Each axis can be logged by providing a data axis config for it.